templates/zerver/integrations.html

```diff
@@ -1830,6 +1830,42 @@
       <img class="screenshot" src="/static/images/integrations/stash/005.png" />
     </div>
 
+    <div id="stripe" class = "integration-instructions">
+
+      <p>You can choose to be notified whenever certain events are triggered on Stripe
+      by adding a webhook to your Stripe account. First, create a stream where the events
+      are to be sent.</p>
+
+      <p>Next, on your {{ settings_html|safe }}, create a Stripe bot.</p>
+
+      <p>Add a webhook to your Stripe account by going to "Your account"->"Account settings"->"Webhooks"->"Add Endpoint"
+         and adding the following URL, replacing the bot email address(with the <code>@</code>sign escaped to <code>%40</code>), bot API key, and Zulip stream with the appropriate
+         values: </p>
+      <p><code>{{ external_uri_scheme }}<font color="#00A26F">stripe-bot%40example.com</font>:<font color="#00A26F">api_key</font>@{{ external_api_path_subdomain }}/v1/external/stripe?stream=<font color="#00A26F">sales</font></code></p>
```

Comment:
FYI stripe is special and will only send webhooks to https sites (since it is a payment platform).

I recommend
1. Change the image to use https in the example.
2. Use https only in the examples
3. Condition on the external_url_scheme and if it is http warn that Stripe only works with an https url.